pandas/tests/test_panel.py

```diff
@@ -1143,23 +1143,11 @@ def test_from_dict_mixed_orient(self):
         assert panel['A'].values.dtype == np.float64
 
     def test_constructor_error_msgs(self):
-        msg = (r"Shape of passed values is \(3, 4, 5\), "
-               r"indices imply \(4, 5, 5\)")
+        msg = (r"Shape of passed values is \(4, 5, 3\), "
+               r"indices imply \(5, 6, 4\)")
         with pytest.raises(ValueError, match=msg):
             Panel(np.random.randn(3, 4, 5),
-                  lrange(4), lrange(5), lrange(5))
-
```

Comment:
leave these tests here

Reply:
okay. Is there additional value in those tests?